mattlianje/gregarious-wolf
# Created Pitch Analysis

- Outputs Pitch graph and data for given audio

- Changed TESTING mode so it doesn't initiate sql connection

- Made amplitude analysis catch the last second of a file (was previously ignoring it lol)

**HOW TO TEST**

- `cd audio-analysis`
- run `python main.py log`
- choose any option, pitch analysis should be run along with amplitude

_Note: Pitch is not output in Hertz but as a specific tonal value (Mel's scale)_
